backend/composer/services/cs_ingestion/helpers/overwritable_helper.py

```diff
@@ -10,25 +10,35 @@
 logger_service = LoggerService()
 
 
-def get_overwritable_statements(statements_list: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
-    overwritable_statements = [
+def get_overwritable_and_new_statements(statements_list: List[Dict[str, Any]], disable_overwrite: bool=False) -> List[Dict[str, Any]]:
+    overwritable_and_new_statements = [
         statement for statement in statements_list
-        if not has_invalid_sentence(statement) and not has_invalid_statement(statement)
+        if not has_invalid_sentence(statement, disable_overwrite) and not has_invalid_statement(statement, disable_overwrite)
```

Comment:
suggestion: I think we should rename the has_invalid_x to is_new_or_overwritable_x (this will reverse the logic and remove the nots, making it more readable). 

Whoever came up with the has_invalid name should rethink their life decisions (it was me 😛)

Reply:
Great point Afonso. It helped a lot.

Reply:
Thanks. Reads much better now 💯